src/server/pegasus_server_write.cpp

```diff
@@ -51,11 +58,20 @@ int pegasus_server_write::on_batched_write_requests(dsn::message_ex **requests,
         return _write_svc->empty_put(_decree);
     }
 
-    auto iter = _non_batch_write_handlers.find(requests[0]->rpc_code());
-    if (iter != _non_batch_write_handlers.end()) {
-        dassert_f(count == 1, "count = {}", count);
-        return iter->second(requests[0]);
+    try {
```

Comment:
Why use `try ... catch ..` rather than `if ... else ...`

Reply:
What to do here with `if ... else ...`?